README.md

```diff
@@ -1 +1,83 @@
-# Metrics-webapp
\ No newline at end of file
+### Metrics webapp
+
+Metrics webapp is a React-Redux application consists of two pages, first oage for all contries, second page is about details of selected country . it that allows you to:
+
+- See different flags .
+- See different details like {population , location, state, ....}.
+
+## Live virsion 
+[Live link](https://62e1af14570c7107185ee91d--chic-pixie-431455.netlify.app/)
```

Comment:
- [ ] We need to ensure the app is properly deployed. The filter dropdown is not present in the live demo link provided. Kindly re-deploy your app to have this change.

  Live link (no filter) | Local app
  :---:|:---:
  ![op-1](https://user-images.githubusercontent.com/54848363/181589459-159f1fe8-2960-4180-acf0-68df9495b378.jpg) | ![op-2](https://user-images.githubusercontent.com/54848363/181589438-34d61a2f-d98d-4db8-b6cb-7e45bb9db2f9.jpg)